pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -36,6 +37,12 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/ovs/ovsconfig"
 )
 
+// NetDeviceType type Enum
+const (
+	veth = "veth"
```

Comment:
please consider a longer name so that they are easier to localize, e.g. `netDeviceTypeVeth` and `netDeviceTypeVf`